main/lsp/LSPTypechecker.cc

```diff
@@ -423,7 +521,12 @@ void LSPTypechecker::commitTypecheckRun(TypecheckRun run) {
         if (id >= indexed.size()) {
             indexed.resize(id + 1);
             globalStateHashes.resize(id + 1);
-            diagnosticEpochs.resize(id + 1);
+            // No diagnostics sent yet; initialize new IDs to 0.
+            diagnosticEpochs.resize(id + 1, 0);
```

Comment:
Primitive values aren't auto-initialized to 0, so I realized I should probably fix that here to avoid garbage epoch values.